shared/rocroller/test/catch/ExpressionTransformationTest.cpp

```diff
@@ -430,6 +430,66 @@ TEST_CASE("FastArithmetic includes translate time evaluation",
     CHECK_THAT(fastArith(c * zero), IdenticalTo(literal(0.f)));
 }
 
+TEST_CASE("FastArithmetic pipeline properly simplifies expressions",
+          "[expression][expression-transformation]")
+{
+    using namespace rocRoller;
+    using Expression::literal;
+    auto context = TestContext::ForDefaultTarget();
+
+    Expression::FastArithmetic fastArith(context.get());
+    auto                       transforms = fastArith.getTransforms();
+
+    auto isSimplify = [](const auto& transform) {
+        auto* func
+            = transform.template target<Expression::ExpressionPtr (*)(Expression::ExpressionPtr)>();
+        return func && *func == Expression::simplify;
+    };
```

Comment:
minor: let's help the c++ parser.
```suggestion
    auto isSimplify = [](const ExprTransformType& transformFunction) {
        using ExprTransformFuncPtrType = ExpressionPtr(*)(ExpressionPtr);
        const auto* funcPtr = transformFunction.target<ExprTransformFuncPtrType>();
        return funcPtr && *funcPtr == simplify;
    };
```

Reply:
Added in 4c4215d